android/src/main/java/ly/count/android/sdk/react/CountlyReactNative.java

```diff
@@ -749,6 +749,28 @@ public void event(ReadableArray args) {
         }
     }
 
+    @ReactMethod
+    public void recordEvent(ReadableMap args) {
+        String eventName = args.getString("n");
+
+        int eventCount = 0;
+        if (args.hasKey("c")) {
+            eventCount = args.getInt("c");
+        }
+
+        double eventSum = 0;
```

Comment:
maybe for clarity, we could provide 0.0 by default

Reply:
done